test/test_suite_test.go

```diff
@@ -48,9 +48,9 @@ func init() {
 		os.Exit(-1)
 	}
 
-	log.SetOutput(GinkgoWriter)
-	log.SetLevel(log.DebugLevel)
-	log.SetFormatter(&log.TextFormatter{
+	logrus.SetOutput(GinkgoWriter)
+	logrus.SetLevel(logrus.DebugLevel)
+	logrus.SetFormatter(&logrus.TextFormatter{
```

Comment:
Of these 3, SetOutput and SetFormatter are only possible on the logrus package (https://github.com/cilium/cilium/commit/ca0dbdf96e6f7331df091e7ffbfe622b521f220e#diff-599b7f4143f4fea0dcb52272ada9be3aR110) but SetLevel can be called on the log object. Calling it on the logrus package logger changes the log level for the same logger used by libraries, and they are noisy in debug. I'd suggest making a package variable called log (`log = common.DefaultLogger` is how I was doing it elsewhere) and then call SetLevel on it directly. The other places we try to log would then keep using `log.`.
You might want to suppress more of the logrus default logger messages by changing that level (see https://github.com/cilium/cilium/commit/ca0dbdf96e6f7331df091e7ffbfe622b521f220e#diff-599b7f4143f4fea0dcb52272ada9be3aR121)